src/plugins.js

```diff
@@ -0,0 +1,117 @@
+'use strict'
+
+const { app, nativeImage } = require('electron')
+const { join } = require('path')
+const { warn, verbose, logger } = require('./common/log')
+const { uniq, pluck } = require('./common/util')
+const { promises: jsonld } = require('jsonld')
+
+
+class Plugins {
+  constructor(root, config) {
+    this.root = root || ARGS.home || app.getPath('userData')
+    this.config = config || this._loadConfig() || []
+    this.instances = []
+  }
+
+  _loadConfig() {
```

Comment:
We don't use underscore to mark 'private' methods anywhere. If you really want a private instance methods you can define them using a WeakMap or you can make it stateless helper function which you don't export (would just have to pass in `root` in this case).

Reply:
Yeah, I was no longer sure about the underscore there, will just get rid of it.